package.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Don't make the empty test, if there are no tests to run here, that's useful information that something like this would hide.  Go ahead and remove the package.json and package-lock.json files for now along with the `test-js` makefile target for now.